diagrams/WatchCommand.puml

```diff
@@ -0,0 +1,76 @@
+@startuml
+
+!include style.puml
+box
+participant ":InputParser" as InputParser LOGIC_COLOR
+participant ":WatchCommand" as WatchCommand LOGIC_COLOR
+participant ":UI" as UI LOGIC_COLOR
+participant ":WatchTime" as WatchTime LOGIC_COLOR
+participant ":ShowList" as ShowList LOGIC_COLOR
+end box
+
+[-> InputParser : watch("friends")
+activate InputParser
+
+InputParser -> WatchCommand: showWatched("friends")
+activate WatchCommand
+
+WatchCommand -> WatchCommand : processCommand()
+activate WatchCommand
+
+
+
+alt In middle of series
+WatchCommand -> ShowList : increment currentEpisode
+activate ShowList
+ShowList--> WatchCommand
+deactivate ShowList
+
+WatchCommand -> UI : printChangeEpisode()
+activate UI
+UI --> WatchCommand :
+
```

Comment:
<img width="628" alt="1" src="https://user-images.githubusercontent.com/60425261/97667088-0e3c3c80-1aba-11eb-84be-bb9b5713b2af.PNG">
This should be a dotted line. Seems like  you guys did use the --> . Maybe can try UI --> WatchCommand : printChangeEpisode()